packages/create-vite/build.config.ts

```diff
@@ -28,8 +28,131 @@ export default defineBuildConfig({
           path.resolve(__dirname, './LICENSE'),
           'create-vite license',
           'create-vite',
+          '# License of the files in the directories starting with "template-" in create-vite\n' +
+            'The files in the directories starting with "template-" in create-vite are licensed under the CC0 1.0 Universal license:\n\n' +
```

Comment:
```suggestion
          '# License of the files in the directories starting with "template-" in create-vite\n' +
            'The files in the directories starting with "template-" in create-vite and files generated by create-vite are licensed under the CC0 1.0 Universal license:\n\n' +
```



Perhaps we should also mention the files generated as external users may not know we generate templates based on `template-*`.